crates/commonware-node/src/dkg/manager/actor/mod.rs

```diff
@@ -105,31 +85,27 @@ where
     ) -> eyre::Result<Self> {
         let context = ContextCell::new(context);
 
-        let ceremony_metadata = Metadata::init(
-            context.with_label("ceremony_metadata"),
-            commonware_storage::metadata::Config {
-                partition: format!("{}_ceremony", config.partition_prefix),
-                codec_config: (),
+        // Initialize the unified metadata database
+        let metadata: Metadata<ContextCell<TContext>, FixedBytes<32>, Bytes> = Metadata::init(
+            context.with_label("dkg_db"),
+            MetadataConfig {
+                partition: format!("{}_dkg_db", config.partition_prefix),
```

Comment:
Similar reasoning - the prefix is already `*_dkg_manager`:

```suggestion
                partition: format!("{}_database", config.partition_prefix),
```